### DISCUSSION TITLE
PHPMailer class not found

### DISCUSSION THREAD

**Discussion by swankier1998** [PHPMailer/PHPMailer#2657] [Q&A]
Hi everyone, can help me please? I try to send a email but there is a error about PHPMailer is not found I try with composer but didn't work so I put the documents in my projet folder, nothing happend and finally I try to use a differents version, 5.2 and 8.1 and nothing again, is my first time used PHPMailer I don't know what a doing wrong. Please help me. Thank for you attention

**Comment by Synchro** [comment]
You need to be much more specific. We have no way of telling what you're doing wrong without details.

I can say that you should definitely not use the old, unsupported 5.2 version on PHP 8.1.

Beyond that, follow the loading instructions in the readme.

**Reply by swankier1998** [reply]
Don't worry I resolved it I didn't include `use PHPMailer\PHPMailer\PHPMailer;
use PHPMailer\PHPMailer\SMTP;
use PHPMailer\PHPMailer\Exception;` because I think with the `require 'vendor/autoload.php';` was enough but apparently wasn't
Greetings

**Reply by Synchro** [reply]
Yes – those `use` statements are only indirectly associated with the class loading, you either need to have them, or use fully-qualified class names in your code.
